python/app.py

```
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", __name__)
+import sys
+import dj_database_url
+from django.conf.urls import url
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+DEBUG = True
+SECRET_KEY = '__secret_key__'
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)